docs/modules/ROOT/pages/explanation/synthesize.adoc

```diff
@@ -0,0 +1,122 @@
+= Synthesize a cluster
+
+include::partial$synthesize-macros.adoc[]
+
+This explanation goes into the details that are omitted in xref:how-tos/synthesize.adoc[synthesizing a cluster].
+
+== Creating a new cluster on control.vshn.net
+
+There's quite a few fields to fill out when creating a cluster.
+
+ID:: The cluster ID is auto-generated if the field is left empty.
+Optionally, you can provide a unique ID for the cluster.
+If the field is filled in, the value will override the auto-generated ID.
+This field is limited to lower-case characters `a-z`, numbers `0-9` and `-`.
+The length is limited to 1-61 characters.
```

Comment:
I wonder: Wouldn't it be better to document the various fields more extensively directly in the portal? Maybe not in a tooltip, but an actual "help" or "description" toggle.

Otherwise we risk documenting stuff twice. resp. one has to fill the webform and switch tabs to this documentation constantly. It's better UX to have the help in the portal while one is operating with the portal.

Reply:
I've got no hard feelings either way

Reply:
We could use this section of the page for a PORTAL ticket later on.